Dockerfile

```diff
@@ -28,6 +27,7 @@ COPY package.json \
 
 # Install dependencies
 FROM base AS build
+RUN apk add --no-cache linux-headers eudev-dev libusb-dev
```

Comment:
This is the node-hid related fix necessary to fix the breaking builds for bledger (cc @nodar-chkuaselidze)

Reply:
Add as comment in file

Reply:
👍